releasenotes/notes/1.0/prepare-1.0-e13b9b8273055775.yaml

```diff
@@ -0,0 +1,71 @@
+  in previous releases <link to 0.44, 0.45, and blog post>,  with the 1.0 release we're
+  completing a transition to have a single ``qiskit`` package exposing only the
+  core SDK (what was previously ``qiskit-terra``). This means that for releases >= 1.0.0,
+  we have stopped using the ``qiskit-terra`` package and will only be publishing ``qiskit``.
+  Unfortunately, this change in packaging structure is not something that
+  can be fully communicated to ``pip``, the Python package installer. This
+  means that it is not possible to just use ``pip install -U qiskit`` to upgrade
+  from qiskit 0.46.0 to 1.0.0. It is strongly recommended that to upgrade to
+  Qiskit 1.0.0 you create a separate virtual environment to isolate the installation. There is a
+  detailed migration guide that explains the packaging changes and how to install
+  Qiskit 1.0.0 in different scenarios which can be found here:
+
+  https://qisk.it/1-0-packaging-migration
+
+  As with all of our major releases, Qiskit 1.0.0 also has a plethora of new
+  features, the highlights for this release are:
+
+    * The :class:`.QuantumCircuit` class's internal :attr:`~.QuantumCircuit.data`
+      structure has been rewritten in Rust to greatly improve the memory efficiency
+      of the :class:`.QuantumCircuit` objects by caching the arguments of the
+      instructions.
+
+    * A new version of the primitives interface definition in :mod:`qiskit.primitives`
+      with the :class:`.BaseSamplerV2` and :class:`.BaseEstimatorV2` abstract
+      classes. This new version of the interface adds support for
+      performing vectorized calls to the primitive so that sweeps over
+      parameter value sets and observables can be efficiently specified.
+
+    * A new experimental native OpenQASM 3 parser :func:`qiskit.qasm3.loads_experimental`
+      and :func:`qiskit.qasm3.load_experimental`. This new parser is still under development
+      and still has several limitations and is still experimental. However,
+      for where it is usable, the new parser is significantly faster and has
+      better diagnostic error message that allows to debug where an OpenQASM 3 program
+      is invalid. This new parser is written in Rust and based on a newly developed
+      Rust library for parsing OpenQASM3 which can be found here:
+      https://github.com/Qiskit/openqasm3_parser
+
+  Finally, the Qiskit 1.0.0 release was an opportunity to clean up some technical debt
+  accumulated over the past 7 years of development. You'll notice that the Qiskit 1.0.0
+  release is virtually free of ``DeprecationWarning``\s being emitted, and, at the same time,
+  the upgrade section of the 1.0.0 release notes is longer than usual. These are direct
+  consequences of the technical debt cleanup. To help with the migration
+  from 0.46.x to 1.0.0, besides the release notes, we have also published a
+  dedicated migration guide which can be found here: **INSERT LINK** . This guide is meant to
```

Comment:
Also don't forget to link here.

Reply:
Done in: https://github.com/Qiskit/qiskit/pull/11760/commits/cfe763cf9bdd677c1c195ff911883385f7b6c417